ds3-sdk-integration/src/test/java/com/spectralogic/ds3client/integration/PutJobManagement_Test.java

```diff
@@ -1103,6 +1120,39 @@ public void objectCompleted(final String name) {
     }
 
     @Test
+    public void testWriteJobWithZeroLengthObject() throws Exception {
+
+        final Ds3Client client = Ds3ClientBuilder.fromEnv()
```

Comment:
Why not use the client that's already created for the integration tests?

Reply:
Will update.